network_importer/adapters/nautobot_api/models.py

```diff
@@ -519,8 +519,13 @@ def translate_attrs_for_nautobot(self, attrs):
                     )
                     continue
 
-                tag_id = device.get_device_tag_id()
-                nb_params["tags"].append(tag_id)
+                try:
+                    tag_id = device.get_device_tag_id()
+                    if tag_id:
+                        nb_params["tags"].append(tag_id)
+                except pynautobot.core.query.RequestError:
+                    # If there is an error with tag retrieval, ignore tags
+                    pass
```

Comment:
shouldn't we log something? (even for DEBUG)

Reply:
@chadell Good call.  I added a warning message